salt/renderers/gpg.py

```diff
@@ -51,6 +51,13 @@
     increasing the system entropy. On virtualised Linux systems, this can often
     be achieved by installing the ``rng-tools`` package.
 
+.. note::
+    As of Ubuntu 18.04 LTS you cannot create a key without a passphrase using
```

Comment:
First of all thanks for the contribution! We love it when more clarification is added to the docs.

Just want to clarifying one thing. Is it only ubuntu 18.04 thats affected or a certain version of gpg? I think it would be more useful to call it out if its a version of gpg that has this requirement or possibly document the error that you were receiving that lead you to realize it required a passphrase.

Reply:
@ksoviero bump

Reply:
It looks it depends on gpg version. There are a number of references to 2.0 and 2.1 changes of the related behavior in man page. I'm not sure what's exactly. But what I'd like to suggest here is to not add any not. Just change the original `gpg` command to work as expected with any version. Maybe this one would be good enough?
```
gpg --quick-gen-key --batch --homedir /etc/salt/gpgkeys --passphrase '' <username>
```

Reply:
@saltstack/team-core thoughts?

Reply:
im good with that approach